core/impl/sf6/track.go

```diff
@@ -110,13 +109,18 @@ func (t *SF6Tracker) Start(userCode string, restoreData bool, refreshInterval ti
 	}
 
 	if !restoreData {
-		t.refreshMatchHistory(battleLog)
+		t.state = t.getNewTrackingState(battleLog)
 	}
 
 	log.Println(`Profile loaded `)
 	t.isTracking = true
+
+	err := t.TrackerRepository.SaveUser(t.ctx, t.state.CFN, t.state.UserCode)
+	if err != nil {
+		log.Fatalf("save user: %v", err)
```

Comment:
Should we really exit here, wouldn't the tracker continue to work for the current session? If so then could just display an error message to user